src/sparql/update/insert_emc_synonym_provenance.ru

```diff
@@ -1,15 +1,14 @@
 PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
-PREFIX mondo: <http://purl.obolibrary.org/obo/mondo#>
-prefix owl: <http://www.w3.org/2002/07/owl#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
 PREFIX OMO: <http://purl.obolibrary.org/obo/OMO_>
 
 INSERT {
   ?syn_annot oboInOwl:hasDbXref ?xref_value .
 }
 WHERE {
     VALUES ?syn_type {  
-      <https://w3id.org/information-resource-registry/nord>
-      <https://w3id.org/information-resource-registry/clingen>
+      "https://w3id.org/information-resource-registry/nord"
+      "https://w3id.org/information-resource-registry/clingen"
```

Comment:
This change looks odd to me, do you know why this was needed? Does this surely work now?

Reply:
> PS - some of these queries with this new property only work in the OWL version. I added a comment into the other query where this happens.

This is super weird and suggests a bug in the OWL API.. Thanks for uncovering